examples/platforms/efr32mg13/sleepy-demo/README.md

```diff
@@ -1,6 +1,6 @@
 # EFR32MG13 Sleepy Demo Example
 
-The EFR32 Sleepy applications demonstrates Sleepy End Device behaviour using the EFR32's low power EM2 mode. The steps below will take you through the process of building and running the demo
```

Comment:
Please m ake all the same changes to the efr32mg12 and efr32mg21 versions of this file.